Library/Formula/ice.rb

```diff
@@ -1,38 +1,26 @@
-require 'formula'
-
 class Ice < Formula
-  desc "Distributed computing platform"
-  homepage 'http://www.zeroc.com'
-  url 'http://www.zeroc.com/download/Ice/3.5/Ice-3.5.1.tar.gz'
-  sha1 '63599ea22a1e9638a49356682c9e516b7c2c454f'
-  revision 1
+  desc "A comprehensive RPC framework"
+  homepage "https://zeroc.com"
+  url "https://github.com/zeroc-ice/ice/archive/v3.6.0.tar.gz"
+  sha256 "77933580cdc7fade0ebfce517935819e9eef5fc6b9e3f4143b07404daf54e25e"
 
   bottle do
     sha1 "52d43a08202206258c6fa33750aac25d2dc38703" => :yosemite
-    sha1 "df1279e47c8c213a57d04318443605b2871269a7" => :mavericks
-    sha1 "7bdb347052b8df900c2c2e6797726a79d9c0242d" => :mountain_lion
   end
 
-  option 'doc', 'Install documentation'
-  option 'demo', 'Build demos'
+  option "without-java", "Compile without Java support."
 
-  resource "berkeley-db" do
-    url "http://download.oracle.com/berkeley-db/db-5.3.28.tar.gz"
-    sha1 "fa3f8a41ad5101f43d08bc0efb6241c9b6fc1ae9"
-  end
-
-  depends_on "openssl"
+  depends_on :macos => :yosemite
   depends_on "mcpp"
-  depends_on :python => :optional
+  depends_on :java => "1.7" if build.with? "java"
```

Comment:
Java 7 is required to build Berkeley DB. This was brought up a few months ago, and as of then there is no way to patch a vendored part of a formula.